db/database.go

```diff
@@ -100,3 +109,65 @@ func (db *Database) ClearAllOffers() error {
 
 	return nil
 }
+
+// PutSwap puts the given swap in the database.
```

Comment:
Can we document and maybe even unit test what happens if `s.ID` is already in the database? Does it generate an error or does it overwrite the existing entry?

Reply:
done